tabs/src/tabs/save.js

```diff
@@ -41,7 +41,23 @@ export default function save( { attributes: { tabs } } ) {
 
 	return (
 		<div { ...blockProps }>
-			<div role="tablist">{ tabButtons }</div>
+			<div className="tabs-container">
+				<div
+					className="scroll-arrow scroll-arrow-left"
```

Comment:
We are not using regular button elements here, as the VIP simple sites try to filter this during frontend rendering.